### DISCUSSION TITLE
Could we make list of bitbucket repositories configurable ?

### DISCUSSION THREAD

**Discussion by mhornspe** [reywood/bitbucket-pr-commit-message#2] [Q&A] [answered]
Hi !

This extension is exactly what I would need.
Our team does not use bitbucket.org but rather an internally hosted version of bitbucket, hence having another URL.
Would it be possible to make the list of bitbucket repositories configurable ?

Thank you for this great initiative
Mathieu

**Comment by reywood** [comment, marked as answer]
Hi there. I think there are a couple of problems that prevent this.

The first is that every browser extension has to specify what websites it will be active on. This is done in a static manifest file that I don't believe is configurable after the extension is installed. So I'd have to give the extension access to all websites which would be a red flag for a lot of people.

The second problem has to do with differences in the underlying page structure of privately hosted BitBucket instances vs the bitbucket.com website. I don't think they are the same, so there's a good chance that the extension wouldn't work even if I did give it global access.

I think your best bet is to copy the code for this extension to your local machine, modify the manifest to work with your private instance, and then [install it in developer mode](https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked) to see if it works. If not, you'll have to play with the element selectors in the code to get it to work.

Hope that helps and sorry for the delay in getting back to you.

**Reply by mhornspe** [reply]
Thank you for your reply. I was not aware of all these technical challenges.
